exonum-java-binding-core/rust/generate_licenses.sh

```diff
@@ -1,7 +1,17 @@
 #!/usr/bin/env bash
 # Gather licenses for every Rust dependency and print it in the file in a readable form.
 
-cargo license -h || cargo install cargo-license
+cargo license -h &> /dev/null || cargo install cargo-license
 
-cargo license -da | sed 's@registry+https://github.com/rust-lang/crates.io-index, @@g' | sed 's/\x1b\[[0-9;]*m//g' > target/LICENSES-THIRD-PARTY-NATIVE
+# Going to exonum-java-binding-core/rust directory.
+backup_dir=${PWD}
+script_dir=$(dirname $(readlink -f "$0"))
+cd $(dirname $0)
 
+cargo license -da \
+| sed 's@registry+https://github.com/rust-lang/crates.io-index, @@g' \
+| sed 's/\x1b\[[0-9;]*m//g' \
```

Comment:
I’d document these steps so that it’s clear why they are needed.

Reply:
Done